libs/ballot-interpreter-nh-next/src/interpret.rs

```diff
@@ -349,20 +349,15 @@ pub fn interpret_ballot_card(side_a_path: &Path, side_b_path: &Path, options: &O
 
 #[cfg(test)]
 mod test {
-    use std::{fs::File, io::BufReader};
+    use std::{fs::File, io::BufReader, path::PathBuf};
 
     use crate::ballot_card::load_oval_template;
 
     use super::*;
 
     #[test]
     fn test_interpret_ballot_card() {
-        let fixture_path = Path::new(file!())
-            .parent()
-            .unwrap()
-            .parent()
-            .unwrap()
-            .join("test/fixtures/ashland");
+        let fixture_path = PathBuf::from(env!("CARGO_MANIFEST_DIR")).join("test/fixtures/ashland");
```

Comment:
This change was because the `file!()` macro gives a path relative to the workspace root when using workspaces: https://stackoverflow.com/questions/30003921/how-can-i-locate-resources-for-testing-with-cargo.